File: webservices/src/main/java/fieg/modulos/cobrancas/dto/CobrancasDTO.java
package fieg.modulos.cobrancas.dto;

import lombok.Data;

import java.time.LocalDate;

@Data
public class CobrancasDTO {

    private String entidade;
    private String codUnidade;
    private String unidade;
    private String filial;
    private String responsavelFinanceiro;
    private String cpfCnpj;
    private LocalDate nascimento;
    private String consumidor;
    private Integer idSistema;
    private String sistema;
    private Integer ano;
    private String produtoServico;
    private String protheus;
    private String proposta;
    private Integer contrato;
    private Integer idCobrancaAutomatica;
    private Integer idProduto;
    private Integer idOperadorConsultor;
    private String consultor;
}

File: web/src/models/DTOs/Cobrancas/CobrancasDTO.ts
export interface CobrancasDTO {
  entidade: string;
  codUnidade: string;
  unidade: string;
  filial: string;
  responsavelFinanceiro: string;
  cpfCnpj: string;
  nascimento: number;
  consumidor: string;
  idSistema: number;
  sistema: string;
  ano: number;
  produtoServico: string;
  protheus: string;
  proposta: string | null;
  contrato: number;
  idCobrancaAutomatica: number | null;
  idProduto: number;
  idOperadorConsultor: number | null;
  consultor: string | null;
}
